hedgehog/src/Hedgehog/Internal/Prelude.hs

```diff
@@ -8,10 +8,15 @@ module Hedgehog.Internal.Prelude (
     Semigroup(..)
   , MonadFail
   , module Prelude
+  , (...)
   ) where
 
 import           Control.Monad.Fail (MonadFail)
 
 import           Data.Semigroup (Semigroup(..))
 
 import           Prelude hiding (filter, print, map)
+
+(...) :: (c -> d) -> (a -> b -> c) -> a -> b -> d
+(...) = (.) . (.)
+{-# INLINE (...) #-}
```

Comment:
I think I'd rather not have this, this module was more intended for compatibility than introducing custom operators. I'd like to keep things as vanilla as possible for now.

Maybe you can convince me, but it shouldn't be part of this PR.

Reply:
@HuwCampbell, perhaps also we can move this to the let binding above where it’s used?

This clashes with ES6 spread syntax, so perhaps we could just rewrite it without (...) altogether.

Reply:
Gone.  I've just eta expanded both arguments.

I do like the owl (called so as its implementation is `(.) . (.)`) as it's just another simple form of function composition, but I understand what you're going for with this module.